polkadot/runtime/common/src/impls.rs

```diff
@@ -371,6 +393,46 @@ mod tests {
 		t.into()
 	}
 
+	pub fn deprecated_era_payout(
```

Comment:
Since this fn is not used anywhere but in the tests, does it need `#[allow(dead_code)]` to avoid compilation warnings?

I'd probably have moved this implementation and the tests into a `mod test_inflation {}` as we probably dont use this code in the runtime anymore.

Reply:
nit but moving this code under a mod gated by cfg(test) also make sense given that in the prdoc, it says that the old era payout impl has been removed :)

Reply:
Yes, it currently lives under 
```
#[cfg(test)]
mod tests {
```
in L265